gen/plugins_with_define/com.salesforce/com.salesforce.plugin.smartsync.js

```diff
@@ -30,79 +30,89 @@ var SALESFORCE_MOBILE_SDK_VERSION = "5.0.0";
 var SERVICE = "com.salesforce.smartsync";
 
 var exec = require("com.salesforce.util.exec").exec;
+var defaultStoreName = "defaultStore";
```

Comment:
This has to match what's defined in `native`, correct? Can we remove this, so `native` is the single source of truth and we don't have to maintain them in both?